tests/dummy/app/templates/docs/sprites.md

```diff
@@ -0,0 +1,68 @@
+# Sprites
+
+## What is a sprite?
+
+A sprite is an html element that can animate by using its starting and ending positions. The term "sprite" comes from a long tradition of two-dimensional graphics. In ember-animated, sprites carry additional data about themselves. This extra information is typically the initial and/or the final location of the sprite. Sprites can animate from their starting position to their final position and vice versa. 
```

Comment:
One writing pattern I like to follow is to follow a "why, how, what" progression. Why is this needed/what problem does it solve, how does it solve it, and what do you do. Here's an example reordering using this pattern:

**Why:** Sprites are useful for animating html elements as they are inserted, removed, or moved around to new positions. For example, imagine that a new email has just arrived. Where should it show up within a list of other emails? How should things move and shift when the user deletes an email, or maybe multiple emails at once? Sprites help manage these states and create a smooth flow between them. 

**How:** The term "sprite" comes from a long tradition of two-dimensional graphics. However, in ember-animated, a sprite can do a lot more than a regular html element can. It holds data about its own position and state. Animations can be played forwards and backwards, in and out of the user's view. By using sprites, the developer does not need to track the state of different elements by hand.

**What:** Before we can write sprite animations, it's important to understand the lifecycle of a sprite as it enters, leaves, or is moved around. There are five different categories that a sprite could belong to: `insertedSprites`, `keptSprites`, `removedSprites`, `sentSprites`, and `receivedSprites`.  Let's begin with the first three categories.

...etc